package.json

```diff
@@ -19,7 +19,7 @@
     "build": "node ./scripts/build.js",
     "build-storybook": "build-storybook",
     "prepublishOnly": "pinst --disable && node ./scripts/prepublish.js",
-    "postinstall": "husky install",
+    "postinstall": "husky install && cd node_modules/web-components && npm install --only=prod && npm run-script build",
```

Comment:
This is because of how we are distributing our web component library right now (not publishing to npm). We can add the dependency as a git url, but we also need to do a build since the built files aren't tracked. In order to do a build, this will only install the dependencies, not the `devDependencies`.

Reply:
:grimacing: Ugly, but necessary without expanding the scope of this PR, from what I can tell.

A cleaner way might be to use a GH action when the version bumps to
1. Build the library
2. Push the result to a branch
3. Create a new release from that branch which contains _only_ the built code

Let's loop back around to improve the process later, though.

Reply:
I think that would be a good enhancement, because we are going to have to do this same thing in `vets-website`.